ui/components/app/confirm-page-container/confirm-page-container-container.test.js

```diff
@@ -61,6 +61,7 @@ describe('Confirm Page Container Container Test', () => {
     showAddToAddressBookModal: sinon.spy(),
     contact: undefined,
     isOwnedAccount: false,
+    hideTitle: false,
```

Comment:
Why was this change needed? It appears to be unrelated to the other changes.

Reply:
This snuck in from the last rebase.